packages/server/src/app-server.ts

```diff
@@ -36,12 +38,16 @@ function isReachable(port: number, options: { timeout: number } = { timeout: 100
 };
 
 export function* createAppServer(options: AppServerOptions): Operation {
-  yield ChildProcess.spawn(options.command, options.args || [], {
+  let child = yield ChildProcess.spawn(options.command, options.args || [], {
     cwd: options.dir,
     detached: true,
     env: Object.assign({}, process.env, options.env),
   });
 
+  let logStream = createWriteStream(options.logPath);
+  child.stdout.pipe(logStream);
+  child.stderr.pipe(logStream);
```

Comment:
These streams should terminate automatically when the process exits, which should close the pipes.